components/molecule/selectPopover/src/index.js

```diff
@@ -119,7 +125,11 @@ function MoleculeSelectPopover({
         </Button>
       ) : null}
       {cancelButtonText ? (
-        <Button onClick={handleOnCancel} design="flat" {...cancelButtonOptions}>
+        <Button
+          onClick={handleOnClickCancelButton}
+          design="flat"
```

Comment:
In these kind of props I'm trying to use the enum that you can import from the Button component